charter.md

```diff
@@ -2,92 +2,82 @@
 
 ## Introduction
 
-Digital credentials are essential to identity, authorization, licenses, certificates and digitization use cases that are part of modernization efforts targeting efficiency, transparency, such as digital licenses or certificates of origin.
-IETF WGs and IRTF RGs have provided COSE and JOSE based building blocks.  Things like identifiers, claim structures, and confidentiality for personally identified information enable interoperability and cryptographic agility, however, they have required profiling in order to render digitization easy, unambiguous and consistent.
-The many ways to combine identifiers, key material, signatures, and attributes to create credentials, often prevents interoperabtility from spanning across data silos or cross use cases.
-This lack of interoperabilty leads to increased implementation costs, attack surface, and harms adoption, which prevents digitization from being deployed successfully.
-SPICE aligns the capabilities of JOSE and COSE to support modern cryptographic capabilities, such as selective disclosure or unlinkablilty, consistent identity attribute semantics across representations, and Internet-scale discoverability of identifiers and cryptography capabilities.
+Digital credentials may be thought as digital equivalents of paper-based passports, driver's licenses, membership certificates or proofs of qualification attached to a person.
+These types of credentials usually contain personal data such as a family name, first names, a gender, a birth date, an home address, a phone number, a face picture
+or minutiae points for fingerprints.
```

Comment:
```suggestion
Digital credentials are essential to identity, authorization, licenses, certificates and digitization use cases that are part of modernization efforts targeting efficiency, transparency, such as digital licenses or certificates of origin.
```

Reply:
I don't support framing the work as "for natural persons only", I would like to revert this set of changes to what we had previously.

Reply:
I concur that natural persons frames this narrowly in a way I do not think is productive for the charter and this group at this juncture.

Reply:
> I don't support framing the work as "for natural persons only", I would like to revert this set of changes to what we had previously.

However, no rational for this rejection has been provided. So I can only wonder why. 
Is there an intent to address legal persons as well ? 
If it is the case, I wonder why legal persons would have privacy concerns. 
Natural persons have privacy concerns, why would it also apply to legal persons ?
If it is not the case of legal persons, why other case(s) is it ?

Reply:
Digital Credentials may be used to represent assets or organizations, not just natural persons, there should be no need to restrict it to just natural persons

Reply:
Mesur is using digital credentials for many use cases where they do not represent natural persons. Restricting things in this manner is not necessary.